### ISSUE TITLE
A11y_VisualStudioCodeClient_Search_Global Search_ Focus Visible: The keyboard focus is not visible on Case buttons after activating them.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#28904]
-A11yMAS; -A11yTCS; -A11ySev3; -DesktopApp; -Win32; -Visual Studio Code Client; -BM-VisualStudioCodeClient-Win32-Aug2023; -WCAG2.4.7; -AILimited; -FocusVisible; -Keyboard; 

--- Environment and OS details:
Visual Studio Code insider version 1.82.0
Microsoft Windows 11 Enterprise 22H2 Build 22621.1992

--- Repro Steps:
1) Open Visual Studio Code Insiders editor  
2) Using keyboard navigation View Menu Bar
3) Select search option
4) Search with match casing enabled
5) Search with match whole world enabled
6) Search using RegEx
7) Search with file excludes
8) Search with file includes
9) Open search results in the Search Editor

--- Actual Result:
The keyboard focus is not visible on Case buttons after activating them.

--- Expected Result:
Keyboard Focus should be visible after activating the case buttons”.

--- User Impact:
Users who rely on keyboard will be impacted if keyboard focus indicator is not visible for the interactive controls. 

--- Attachment:
